src/behaviors/loginPrompt.js

```diff
@@ -1,35 +1,42 @@
 export default {
-  /**
-   * Init login prompt observer
-   * Called by Polymer on attach
-   * @return {undefined}
-   */
-  attached() {
-    this._promptLoginOnEditable();
+  properties: {
+
+    loginPrompt: {
+      type: Boolean,
+      observer: '_promptLoginOnEditable'
+    }
+
   },
 
   /**
    * Observer to prompt for login if editable && !authed
    * @return {undefined}
    */
-  _promptLoginOnEditable() {
-    let simplaLogin = this.$['login'],
+  _promptLoginOnEditable(loginPrompt) {
```

Comment:
`loginPrompt` is the user configuration, but doesn't need to be the parameter name, no reason the parameter name can't be called something more understandable. If the function was called something like `loginPromptObserver` then having it called the property name would make sense. JSDoc would help, though better if clearer in the comments.